### ISSUE TITLE
Is there code for scattering network coefficients ...

### ISSUE THREAD

**Issue by kmamykin** [RandallBalestriero/fast_scattering_network#1]
Hi, I am researching the scattering networks and came across your paper https://openreview.net/pdf?id=SJiFvr9el 
LINEAR TIME COMPLEXITY DEEP FOURIER SCATTERING NETWORK AND EXTENSION TO NONLINEAR INVARIANTS
which mentions the code in python. 

I am looking for a starter implementation of scattering network algorithm for audio. 
Do you think you will be able to push your code soon? I would love to take a look.

**Comment by RandallBalestriero** [comment]
Hello !
Yes it should be on github (and the paper on arxiv) by the end of next week.
However if you want a basic implementation of the scattering network maybe
the scatnet toolbox available here:
https://github.com/scatnet/scatnet
is a good starting point.

Regards, Randall

**Comment by kmamykin** [comment]
Yes, looked at it. Matlab is not my strong skill, but I get the gist. Looking forward to the code/paper.

Cheers!
